cmd/kubeapply/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/datawire/teleproxy/pkg/kubeapply"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func envBool(name string) bool {
+	val := os.Getenv(name)
+	switch strings.TrimSpace(strings.ToLower(val)) {
+	case "true":
+		return true
+	case "yes":
+		return true
+	case "1":
+		return true
+	case "false":
+		return false
+	case "no":
+		return false
+	case "0":
+		return false
+	case "":
+		return false
+	}
+
+	return true
+}
+
+// Version holds the version of the code. This is intended to be overridden at build time.
+var Version = "(unknown version)"
+
+func adapt(f func(*cobra.Command, []string) error) func(*cobra.Command, []string) {
+	return func(cmd *cobra.Command, args []string) {
+		err := f(cmd, args)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+	}
+}
```

Comment:
`adapt()` is a weird idiom.  What about setting `cmd.SilenceUsage = true` from `PersistenPreRun`? Or even just at the beginning of your `RunE` function?

Reply:
I'm not sure I understand the question. What does SilenceUsage do for me?